packages/nextjs/pages/batches/index.tsx

```diff
@@ -0,0 +1,328 @@
+                            id={`${batch.name}-github`}
+                            href={batch.githubRepoLink}
+                            className="btn btn-xs btn-ghost p-0 min-h-0 w-[24px] h-[24px] hover:opacity-80 flex items-center justify-center align-middle"
+                          >
+                            <Image src="/assets/github-logo.png" alt="GitHub" width={24} height={24} />
+                          </TrackedLink>
+                          {batch.openseaLink && (
+                            <TrackedLink
+                              id={`${batch.name}-opensea`}
+                              href={batch.openseaLink}
+                              className="btn btn-xs btn-ghost p-0 min-h-0 w-[24px] h-[24px] hover:opacity-80 flex items-center justify-center align-middle"
+                            >
+                              <Image src="/assets/opensea-logo.svg" alt="OpenSea" width={24} height={24} />
+                            </TrackedLink>
+                          )}
+                        </div>
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+
+              {/* Pagination */}
+              {totalPages > 1 && (
+                <div className="flex justify-center mt-4">
+                  {Array.from({ length: totalPages }).map((_, index) => (
+                    <button
+                      key={index}
+                      onClick={() => paginate(index + 1)}
+                      className={`mx-1 px-3 py-1 rounded ${
+                        currentPage === index + 1 ? "bg-primary text-white" : "bg-gray-200"
+                      }`}
+                    >
+                      {index + 1}
+                    </button>
+                  ))}
+                </div>
+              )}
+            </div>
+          </div>
+          {/* Next Batch CTA */}
+          <div className="mt-8 card bg-gradient-to-r from-primary to-secondary px-6 lg:pl-6 py-6 max-w-[90%] md:max-w-[75%] xl:max-w-[60%] mx-auto">
+            <div className="card-body p-0 flex flex-col lg:flex-row items-center justify-between">
+              <div className="mb-4 md:mb-0 max-w-full lg:max-w-[55%] text-center lg:text-left">
+                <h3 className="card-title text-2xl text-white mb-2 justify-center lg:justify-start">
+                  Batch #{NEXT_BATCH_NUMBER}
+                </h3>
+                <p className="text-white pr-2">
+                  Complete SpeedRunEthereum and join BuidlGuidl to participate in the next Batch!
+                </p>
+              </div>
+              <TrackedLink
+                id="apply-next-batch"
+                href="https://speedrunethereum.com/"
+                className="btn btn-sm bg-white text-primary hover:bg-gray-100 transition-colors duration-300 inline-flex items-center justify-center whitespace-nowrap mr-10"
+              >
+                Go SpeedRunEthereum
```

Comment:
This button is not centered on mobile screens
![image](https://github.com/user-attachments/assets/37d26eee-50c3-4df1-906a-3ba0c44d8417)

Reply:
Should be fixed on https://github.com/BuidlGuidl/buidlguidl.com/pull/46/commits/0356988189f797ef847010823e36d13660743ded

![image](https://github.com/user-attachments/assets/1b75be9e-03a7-41c3-a30f-e31ae4d85106)